exceptions_tests.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+from pip._vendor.distlib.compat import raw_input
+
+def csv_data_is_supplied_test():
+   try:
+       csv_path_from_user = raw_input("Please insert the csv path: ")
+       test = pd.read_csv(csv_path_from_user)
+       x = test.isnull().any(axis=1)
```

Comment:
``      x = test.isnull().any(axis=1)
 missing_data_rows = x[x == True].index.to_numpy()
       if missing_data_rows.size > 0:
           print("There's some data missing, please check your csv.") 
" 
can be much shoter...

Reply:
done :)